### DISCUSSION TITLE
NODE_ENV is production while in local dev

### DISCUSSION THREAD

**Discussion by jaaneh** [resend/react-email#1673] [General]
Hi,

I was curious if this is a _me_ issue, or if it's an actual issue, but didn't want to open an issue in case it's just me.

When running `email dev` and trying to serve static files from `emails/static` it's still attempting to serve images from our CDN rather than locally.

```js
const isProd = process.env.NODE_ENV === "production"
const BASE_URL = isProd ? "https://cdn.example.com"  : "/static"
const LOGO_URL = `${BASE_URL}/logo.png`
```
```jsonc
// package.json
"@react-email/components": "^0.0.24",
"@react-email/render": "^1.0.1",
"@react-email/tailwind": "^0.1.0",
"react": "^18.3.1",
"react-dom": "^18.3.1",
"react-email": "^3.0.1"
```

This can be a bit annoying when building an email with a new image that only exists locally and not uploaded to CDN yet.

**Comment by gabrielmfern** [comment]
Thinking this is a 3.0 bug, going to try reproducing this myself

**Reply by gabrielmfern** [reply]
Certainly an issue, tried it myself. Made a change and am publishing it, at least on canary, very soon.

**Reply by jaaneh** [reply]
Appreciate it 👏🏼

**Reply by jaaneh** [reply]
Can confirm this is resolved in canary. Thank again!
